lib/internal/backend/file/file.go

```diff
@@ -112,19 +128,82 @@ func (lb *FileBackend) BuildACI(layerIDs []string, dockerURL *types.ParsedDocker
 	return aciLayerPaths, aciManifests, nil
 }
 
-func getImageID(file *os.File, dockerURL *types.ParsedDockerURL) (string, *types.ParsedDockerURL, error) {
+func (lb *FileBackend) BuildACIV22(layerIDs []string, dockerURL *types.ParsedDockerURL, outputDir string, tmpBaseDir string, compression common.Compression) ([]string, []*schema.ImageManifest, error) {
+	if len(layerIDs) < 2 {
+		return nil, nil, fmt.Errorf("insufficient layers for oci image")
```

Comment:
see the above comment "// for oci the first image is the config"

To build an oci we need the sha of the image config object as well as the sha of the layers since it isn't in a known location like it is with v1 docker images. The sha of the image.json is being passed as the first item in the layers slice. It is pulled out below. This was the easiest way to get the data here without modifying the signature of a bunch of methods.